tests/test_visualizer/test_vis_backend.py

```diff
@@ -241,3 +241,54 @@ def test_close(self):
         wandb_vis_backend._init_env()
         wandb_vis_backend.close()
         shutil.rmtree('temp_dir')
+
+
+class TestMLFlowVisBackend:
+    sys.modules['mlflow'] = MagicMock()
```

Comment:
We can add the `mlflow` in requirements/tests.txt and do not need to mock it.

Reply:
reflected.